source/content/partials/traffic-limits-overages.md

```diff
@@ -12,18 +12,18 @@ reviewed: ""
+Pantheon designed Overage Protection for Performance and Elite sites on annual plans to prevent one-time traffic spikes from causing billing issues. All Performance plans and higher (on an annual agreement) include one month of Overage Protection, which provides billing protection against externally driven spikes, or for businesses that have an annual “big event” but otherwise operate at a lower “normal” rate. This protection resets with the calendar year and will be automatically applied the first month of the year your site goes over its plan limits. You as the site owner do not need to do a thing. In a month where Overage protection is applied the site owner will be notified of the event and no other action will be taken. This notice is a great time to review your site's traffic to determine if it is time to move to a new plan and avoid the risk of a future overages bill. Qualified nonprofits receive additional overage protection at no extra cost, automatically applied. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) to learn more about how to qualify for these and other benefits available only to Pantheon's nonprofit customers.
 
-If the change to traffic behavior exceeds your plan limit for any two months of traffic, your site will be moved to the next appropriate plan to help avoid further overages. You will receive notifications of this change in an email with a subject of "You've been right-sized" ahead of time.
+If the change to traffic behavior exceeds your plan limit for a second month in any calendar year, you will receive notice of the traffic volume and the impact of the excess traffic. For online customers (those not on a contract) that will result in an invoice later that same month. For customers on contract overages will be billed quarterly. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) for specifics on what to expect when you receive a notification that your site is over its plan limits.
 
 <Alert title="Note" type="info">
 
-Basic Sites do not have overage protection. If a Basic Site exceeds the 25,000 visit cap in any given month, the site plan will be automatically upgraded to the [Performance plan](https://pantheon.io/plans/performance-pricing) whose visit limit accommodates the site's traffic.
+Sites on a monthly agreement (Basic and Performance) do not have overage protection. If a site on a monthly plan exceeds the published visit cap in any given month, the site plan will be automatically be subject to a bill for the overage. The site owners will be notified of the ent and an invoice will follow later int he same month. 
```

Comment:
Looks like a word got cut off or something: "ent"
@MystikSpiral-74

Reply:
Should be "event".  I'll correct it.